app/Repositories/AlertGroupsRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace App\Repositories;
+
+use Log;
+use App\Repositories\Eloquent\BaseRepository;
+use App\Repositories\Contracts\RepositoryInterface;
+
+class AlertGroupsRepository extends BaseRepository
+{
+    /**
+     * Specify Model class name
+     *
+     * @return mixed
+     */
+    public function setModel()
+    {
+        return 'App\Models\AlertGroup';
+    }
+
+    /**
+     * @param $user_id
+     * @return mixed
+     * get all alert group with user_id
+     */
+    public function findAllAlertGroups($user_id)
```

Comment:
@TranBinhLuan Use `findAll()` instead of `findAllAlertGroups` because it's the function from `AlertGroupsRepository`